src/main/java/co/cask/wrangler/steps/Quantization.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
```

Comment:
2016-2017
Update your copyright profile intellij to have `$today.year` instead of 2016  so that it picks up current year.